### PR TITLE
Feature/sort

### PR DESCRIPTION
**Pull request by joasonvlee**
pls merch

### CODE REVIEW

**Comment by joasonvlee** [comment]
very good

**Comment by joasonvlee** [comment]
pls merch senpai

File: .gitignore [hunk 1]
```diff
@@ -426,6 +426,7 @@ FodyWeavers.xsd
 
 # Custom
 build
+.vscode
```
Comment (djegum) [inline .gitignore:429]: Why?

**Review by djegum — COMMENTED** [review]
I don't know yet

File: .gitignore (hunk 1 above)
Comment (joasonvlee) [reply .gitignore:429]: because reasons

**Review by djegum — APPROVED** [review]
Lets merge!

**Pull request merged**